polaris/__init__.py

```diff
@@ -4,7 +4,16 @@
 from loguru import logger
 
 from ._version import __version__
-from .loader import load_benchmark, load_dataset, load_competition
+from .loader import load_benchmark, load_competition, load_dataset
```

Comment:
Do you aim to organize Python imports alphabetically whenever possible?

Reply:
Hahahaha I once installed a VS Code extension to do it for me. It does satisfy my OCD side! 😅